tests/test_groupsio.py

```diff
@@ -252,11 +258,16 @@ def test_fetch(self):
 
         setup_http_server(empty_mbox=True)
 
-        client = GroupsioClient('beta+api', self.tmp_path, 'aaaaa', verify=False)
+        client = GroupsioClient('beta+api', self.tmp_path, 'jsmith@example.com', 'aaaaa', verify=False)
         success = client.fetch()
 
         # Check requests
         expected = [
+            {
+                'email': ['jsmith@example.com'],
+                'password': ['aaaaa'],
+                'api_key': ['nonce']
```

Comment:
What's `nonce`? According to the post where these breaking changes are described, `api_key` is not needed anymore.

Reply:
I found it here: https://groups.io/api#login-2, but it can be removed since optional